tests/test_polynomes.cpp

```diff
@@ -0,0 +1,120 @@
+/**
```

Comment:
It actually helped a lot for debugging this small class. I would like to create a package at some point to share this polynomial stuff. Potentially in a "control_tools"? Where you would find a PID implementation, the min jerk implementation, potentially a go_to function as well. These kind of things. What do you think?

Reply:
Sounds great, I was already wondering, if it would be possible to generalize the min jerk trajectory such that I could also use it on the higher level :)